src/node/util.ts

```diff
@@ -173,9 +173,10 @@ export const isHashMatch = async (password: string, hash: string) => {
     return false
   }
   try {
-    return await argon2.verify(hash, password)
+    return await verify(hash, password)
   } catch (error: any) {
-    throw new Error(error)
+    logger.error(error)
+    return false
```

Comment:
🧹 I realized it didn't make sense to throw when checking if the hash and password were a match so I refactored to log the error and return `false`.

Reply:
Nice change.